### ISSUE TITLE
old texinfo docs can be removed from current repo

### ISSUE THREAD

**Issue by edhartnett** [Unidata/netcdf-c#1563]
In docs/old we still have the old texinfo docs, assembled by me, last worked on more than a decade ago.

Any differences between the old texinfo docs and the current docs are the result of changes in the library and documentation over time. There is nothing in the texinfo docs that is not in the current docs.

So these can be removed from the current repo. Of course they are always still available in github for historical reasons, even if we remove them from the current code base.

Basically I don't want these old docs to be still around. They are too out of date and cannot do anything but cause confusion.
